src/main/java/mate/controller/driver/GetAllDriversController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet("/drivers")
+public class GetAllDriversController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> allDrivers = driverService.getAll();
+        req.setAttribute("drivers",allDrivers);
+        req.getRequestDispatcher("/WEB-INF/views/drivers/allDrivers.jsp")
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/drivers/all.jsp")
```
і так далі. Просто поділи свої сторінки на пакети до яких вони належать (cars, drivers, manufacturer). для AddDriverTоCar ти можеш створити пакет drivers всередині пакету cars  ---> cars/drivers/add.jsp